modules/util/css-builder/src/main/java/com/liferay/css/builder/CSSBuilder.java

```diff
@@ -61,72 +65,53 @@
 public class CSSBuilder implements AutoCloseable {
 
 	public static void main(String[] args) throws Exception {
-		Map<String, String> arguments = ArgumentsUtil.parseArguments(args);
-
-		List<String> dirNames = new ArrayList<>();
+		Map<String, String> argMap = new HashMap<>();
 
-		String dirName = GetterUtil.getString(
-			arguments.get("sass.dir"), CSSBuilderArgs.DIR_NAME);
+		for (int x = 0; x < args.length; x++) {
```

Comment:
We usually use `i` and `j` for nested loops. But actually, you can do `for (String arg : args)` instead,
 as explained [here](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/for.html).